homeassistant/components/climate/__init__.py

```diff
@@ -132,6 +135,12 @@ async def async_setup(hass: HomeAssistantType, config: ConfigType) -> bool:
         async_service_temperature_set,
         [SUPPORT_TARGET_TEMPERATURE, SUPPORT_TARGET_TEMPERATURE_RANGE],
     )
+    component.async_register_entity_service(
```

Comment:
This PR is changing our entity model. Before we can review this PR there needs to be approval in an issue in our architecture repository.

https://developers.home-assistant.io/docs/core/entity#changing-the-entity-model